health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -431,109 +433,133 @@ const FormulaConfiguration = ({ onSelect, category, customProps, formulas: initi
+                  variation={"primary"}
+                  label={t("NO")}
+                  onClick={() => {
+                    setFormulasPopUp(false);
+                  }}
+                />,
+              ]}
+              sortFooterChildren={true}
+              onClose={() => {
+                setFormulasPopUp(false);
+              }}
+            ></PopUp>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)